tests/ci_tests/code_formatter/crewai/test_crewai_tasks.py

```diff
@@ -0,0 +1,200 @@
+"""Tests for coffee_maker/code_formatter/crewai/tasks.py"""
```

Comment:
Imports are grouped and sorted.
```suggestion
from unittest import mock

import pytest
from crewai import Agent, Task
```